crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -348,6 +349,12 @@ impl Room {
         self.calculate_name().await
     }
 
+    /// Return the last event in this room, if one has been cached during
+    /// sliding sync
+    pub fn latest_event(&self) -> Option<SyncTimelineEvent> {
```

Comment:
Should we feature-gate this method?

```rust
#[cfg(feature = "experimental-sliding-sync")]
pub fn latest_event(&self) -> Option<SyncTimelineEvent> {
```

?

Reply:
Done